includes/admin/settings/register-settings.php

```diff
@@ -1249,19 +1249,19 @@ function edd_header_callback( $args ) {
  *
  * @since 1.0
  * @param array $args Arguments passed by the setting
- * @global $edd_options Array of all the EDD Options
+ *
  * @return void
  */
 function edd_checkbox_callback( $args ) {
-	global $edd_options;
+	$edd_option = edd_get_option( $args['id'] );
 
 	if ( isset( $args['faux'] ) && true === $args['faux'] ) {
 		$name = '';
 	} else {
 		$name = 'name="edd_settings[' . edd_sanitize_key( $args['id'] ) . ']"';
 	}
 
-	$checked = isset( $edd_options[ $args['id'] ] ) ? checked( 1, $edd_options[ $args['id'] ], false ) : '';
+	$checked = isset( $edd_option ) ? checked( 1, $edd_option, false ) : '';
```

Comment:
I just meant we should use `empty()` instead of `isset()`. I wasn't proposing changing `isset( $edd_option )` to `empty( $edd_option )` verbatim.